packages/lib/src/core/core.ts

```diff
@@ -80,6 +80,10 @@ class Core {
 
         this.createCoreModules();
```

Comment:
On this `createCoreModules`, there is a check in place to `console.warn` if modules are already created. I believe we can omit that for PROD version, otherwise merchants will get worried. What do you think?

Reply:
Yes we can enable it only for dev as it doesn't help the merchants.